packages/zowe-explorer/README.md

```diff
@@ -5,6 +5,10 @@
 [![codecov](https://codecov.io/gh/zowe/vscode-extension-for-zowe/branch/main/graph/badge.svg)](https://codecov.io/gh/zowe/vscode-extension-for-zowe)
 [![slack](https://img.shields.io/badge/chat-on%20Slack-blue)](https://slack.openmainframeproject.org/)
 
+> ## :rotating_light: With the removal of keytar shim from VS Code we will be replacing `node-keytar` with `keytar-rs` in our v2.10.0 release to avoid user interuptions
```

Comment:
`interuptions -> interruptions`
`keytar-rs -> @zowe/secrets-for-zowe-sdk`

Reply:
haha don't approve with my misspellings lol
thanks, updates pushed.